ophyd/areadetector/plugins.py

```diff
@@ -101,6 +100,19 @@ def __init__(self, *args, **kwargs):
     def stage(self):
         super().stage()
 
+        if self._misconfigured is None:
```

Comment:
I don't think we do. I'd say 99%+ chance that this detects an ophyd misconfiguration rather than an IOC misconfiguration, meaning that you should be adjusting your startup/detector class and then restarting the session. 

Any strong feelings on this @tacaswell @danielballan ?

Reply:
Usually this requires adjusting the plugins, not rebooting the IOC or restarting the IPython session, but I don't like this idea of the scan sitting here and periodically polling whether the plugins have been fixed.I think it makes sense to keep this as an error.